selendroid-standalone/src/main/java/io/selendroid/standalone/server/SelendroidStandaloneServer.java

```diff
@@ -45,9 +47,10 @@ protected SelendroidStandaloneServer(SelendroidConfiguration config,
     init();
   }
 
-  public SelendroidStandaloneServer(SelendroidConfiguration config)
+  public SelendroidStandaloneServer(SelendroidConfiguration config, InitAndroidDevicesStrategy initAndroidDevicesStrategy)
```

Comment:
Perhaps have a constructor that only takes config and defaults to the old behavior.